cm_service_restart.py

```diff
@@ -56,11 +69,32 @@ def select_from_items(item_type, items):
             service_name = args.service.lower()
         else:
             service_name = select_from_items('service', service_names)
+
+        # Get the action to be performed on predefined actions
+        defined_actions = ['Service restart', 'Deploy Client config', 'Rolling Restart']
+
         for service in services:
             if service.name.lower() == service_name:
 
-                # Restart the service
-                print "Attempting to restart '%s' in '%s'" % (service_name, cluster_name)
-                command = service.restart().wait()
-                print command
-                sys.exit()
+                action = select_from_items('action', [i.lower() for i in defined_actions])
+
+                if action == defined_actions[0].lower():  # Restart the service
+                    command = service.restart().wait()
+
+                elif action == defined_actions[1].lower():    # Deploy client configs
+                    command = service.deploy_client_config().wait()
+
+                elif action == defined_actions[2].lower() and ('hue' in service.name.lower() or 'hive' in service.name.lower() or \
```

Comment:
id rather you get rid of the `if` here with checking the service name against a list like i mentioned below https://github.com/atheiman/cm-automation-scripts-devops-hadoop/pull/2/files#r127274598

but for future reference i think this would work better

```python
elif action == defined_actions[2].lower() and service_name in ['hue', 'hive', 'spark', 'oozie']
    print "Rolling Restart is unavailable in " + service_name
    # ....
elif action == defined_actions[2].lower() and service_name in ['hdfs', 'mapreduce', 'hbase', 'yarn', 'zookeeper']
    print "Rolling Restart is unavailable in " + service_name
```